docs/overview/roadmap.md

```diff
@@ -47,55 +47,34 @@ guidelines to get started.
-  The platform APIs and features are stable and can be depended on for
-  production use. After this plugins will require little to no maintenance.
+  The core of Backstage and its features are stable and can be depended on for production use. After this plugins will require little to no maintenance.
 
-* **[Improved Kubernetes plugin](https://github.com/backstage/backstage/issues/2857)** -
+- **[Improved Kubernetes plugin](https://github.com/backstage/backstage/issues/2857)** -
   Native support for Kubernetes, making it easier for developers to see and
   manage their services running in k8s.
 
-- Further improvements to platform documentation. Examples include a Golden Path
-  for plugin development.
-
-### Future work 🔮
-
-- **Deploy a product demo at `demo.backstage.io`** - Deploy a typical Backstage
-  deployment available publicly so that people can click around and get a feel
-  for the product without having to install anything.
-
-- **[Global search](https://github.com/backstage/backstage/issues/1499)** -
+- **Global search** -
   Extend the basic search available in the Backstage Service Catalog with a
-  global search experience. Long term this search solution should be extensible,
-  making it possible for you add custom search results.
+  global search experience.
 
-- **[[TechDocs V.2] Stabilization release](https://github.com/backstage/backstage/milestone/17)** -
-  Platform stability and compatibility improvements.
+- **[Software Templates V2](https://github.com/backstage/backstage/issues/2771)** -
+  Expanding the templates to make steps more composable, by adding the ability to add more steps for custom logic like webhooks and using authorization from integrations.
 
-- **Additional auth providers** - Backstage should work for most (all!) auth
-  solutions. Since Backstage can be used by companies regardless of what cloud
-  (or on prem) you are using we are especially keen to get auth support for
-  [AWS](https://github.com/backstage/backstage/issues/290),
-  [Azure](https://github.com/backstage/backstage/issues/348) and others.
+### Future work 🔮
+
+- **Golden Path for Plugin Development** -
+  Creating an easy, standardized way for developers to build plugins will encourage contributions and lead to a richer ecosystem for everyone.
 
-- **[Initial GraphQL API](https://github.com/backstage/backstage/milestone/13)** -
+- **[GraphQL API](https://github.com/backstage/backstage/milestone/13)** -
   A GraphQL API will open up the rich metadata provided by Backstage in a single
   query. Plugins can easily query this API as well as extend the model where
   needed.
 
+- **Inter Plugin Communication** -  **[Under consideration]** Establish more clearly defined patterns for plugins to communicate.
+Improved Access Control -   [Under consideration] Provide finer grained access controls and management making it possible for you to better control the platforms user experience. 
```

Comment:
duplicate bullet, hidden under the previous one?

Reply:
Good catch, thanks!